subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/nativeplatform/NativeBuildDependentsPerformanceTest.groovy

```diff
@@ -30,12 +30,16 @@ class NativeBuildDependentsPerformanceTest extends AbstractCrossVersionPerforman
     }
 
     @Unroll
-    def "#task on #testProject"() {
+    def "#task"() {
+        // TODO Enable once runnable on CI (google test & target platform)
+        // 'largeNativeBuild'     | 'project432:buildDependentsExternalComponent111'
+        // TODO Re-evaluate this scenario: memory consumption stress case, gradleOpts = ['-Xms4g', '-Xmx4g']
+        // The generated dependency graph is rather complex and deep, unrealistic?
+        // 'nativeDependentsDeep' | 'libA0:buildDependentsLibA0'
         given:
-        runner.testProject = testProject
         runner.tasksToRun = [ task ]
         runner.args += ["--parallel", "--max-workers=4"]
-        runner.gradleOpts = ["-Xms3g", "-Xmx3g"]
+        runner.gradleOpts = runner.projectMemoryOptions
```

Comment:
Shouldn't this be set automatically?

Reply:
I am not sure where to set this. I actually think the performance test project should define the memory settings in its `gradle.properties`. And the performance test should fail if the test project fails to declare anything. WDYT?